src/widgets/helper/ChannelView.hpp

```diff
@@ -210,10 +210,11 @@ class ChannelView final : public BaseWidget
 
     void drawMessages(QPainter &painter);
     void setSelection(const SelectionItem &start, const SelectionItem &end);
+    void setSelection(const Selection &newSelection);
     void selectWholeMessage(MessageLayout *layout, int &messageIndex);
-    void getWordBounds(MessageLayout *layout,
-                       const MessageLayoutElement *element,
-                       const QPoint &relativePos, int &wordStart, int &wordEnd);
+    std::pair<size_t, size_t> getWordBounds(MessageLayout *layout,
```

Comment:
Some doc comment might be useful.
```suggestion
    /// @returns [wordStart, wordEnd]
    std::pair<size_t, size_t> getWordBounds(MessageLayout *layout,
```

Reply:
comment added ce10b811b024fe625c0f7b8c3f997b7c3aff659f